Adding documentation for new dashboard setting (#4887)
SoftwareAG/c8y-docs: content/cockpit/working-with-dashboards.md

5. Enable the option **Dashboard template** to share the dashboard with all devices of this type. This option is only available for devices dashboards. See [To share a dashboard](#dashboard-template)

> 6. In the **Appearance** tab you can select a theme for the dashboard (one of "Match UI", "Light", "Dark" or "Branded"), a default header style for the widgets (one of "Regular", "Border", "Overlay", or "Hidden") and the gap between the widgets. Moreover, you can select to different layout modes:

Reviewer: ```suggestion
6. In the **Appearance** tab, you can select a theme for the dashboard (one of "Match UI", "Light", "Dark" or "Branded"), a default header style for the widgets (one of "Regular", "Border", "Overlay", or "Hidden"), and the gap between the widgets. Moreover, you can select from two different layout modes:
```

>  - **Grid**: The default one, in which each widget snap into a fixed grid.

Reviewer: ```suggestion
 - **Grid**: The default one, which lets each widget snap into a fixed grid.
```

>  - **Responsive**: Lets the creator define the responsive look of the dashboard. The creator can define the number of columns, select that widgets can be freely positioned (not snapping to the top), or fit the rows into the viewport to create full height.

Reviewer: ```suggestion
 - **Responsive**: Lets you define the responsive look of the dashboard. You can define the number of columns, select that widgets can be freely positioned (not snapping to the top), or fit the rows into the viewport to create full height.
```

>  - **Responsive**: A new option which let the creator define the responsive look of the dashboard. The creator can define the amount of columns, decide that widgets can be free positioned (not snapping to the top) or fit the rows into the viewport to create full-height.

Reviewer: ```suggestion
 - **Responsive**: Lets the creator define the responsive look of the dashboard. The creator can define the number of columns, select that widgets can be freely positioned (not snapping to the top), or fit the rows into the viewport to create full height.
```
